### DISCUSSION TITLE
Parts list updates and alternatives

### DISCUSSION THREAD

**Discussion by sanni** [sanni/cartreader#58] [General Chat]
Found a part cheaper or wanna share a good source? Then post here.

**Comment by HighMans** [comment]
The current BOM has the Microchip IC, 10K and 1K resistors out of stock.

Do the 1k resistors need to be +/- .1% tol? The ebay listing for bulk order shows that they're +/- 5% tol.

For the items not in stock atm, I found these to be cheaper if we ignore the .1% tol.

https://lcsc.com/product-detail/Chip-Resistor-Surface-Mount_YAGEO-AC0805JR-071KL_C140871.html
https://lcsc.com/product-detail/Chip-Resistor-Surface-Mount_YAGEO-RC0805FR-0710KL_C84376.html

**Reply by majorpbx** [reply]
No idea, but I'll tell you I found cheaper parts and they seem to be working.

**Reply by Dakkaron** [reply]
All the resistor values pretty much don't matter. You would not notice a difference if you had 50% tolerance. With one line of code change, you wouldn't even need to have the pulling resistors in there.

**Comment by RJDoran93** [comment]
Has anyone found a suitable replacement or different seller for the N64 Slot (https://www.aliexpress.com/item/33002915153.html) or the 0.96" 4 pin IIC OLED display (https://www.aliexpress.com/item/4000699784061.html). The N64 Slot is no longer available from the linked seller and the OLED Screen seller won't ship to my country (United States). Thank you in advance for any help.

**Reply by Dakkaron** [reply]
The oled can be had on Amazon (though with quickly changing links) for just a few euros more. On Aliexpress it costs €2.20, on Amazon I got it for ~€5-6, but with next day shipping.

**Reply by majorpbx** [reply]
My last order of OLED displays I got [here](https://www.aliexpress.com/item/1005001834365451.html) and you can actually use [these](https://www.aliexpress.com/item/32818485755.html) for N64 and SMS. But you cannot use the N64 slot for SMS. ;)
